homework_4.py

```diff
@@ -0,0 +1,57 @@
+test_info_1 = "Please, write your name: "
+test_info_2 = "Write the letters you want to delete from your name?: "
+test_info_3 = "Your new name is: "
+answer_info_1 = input(f"{test_info_1}")
+answer_info_2 = input(f"{test_info_2}")
+answer_info_3 = answer_info_1.replace(answer_info_2, "")
+result_info = input(f"{test_info_3} {answer_info_3}")
+
+
+example_str = input("Please, enter your sentence: ")
+title_str = example_str.title()
+print(title_str)
```

Comment:
I want to format function for string
print(f"Result is {title_str}") - example

Reply:
fixed